examples/imagenet/readme.md

```diff
@@ -19,12 +19,24 @@ Data Preparation
 
 *By "ImageNet" we here mean the ILSVRC12 challenge, but you can easily train on the whole of ImageNet as well, just with more disk space, and a little longer training time.*
 
-We assume that you already have downloaded the ImageNet training data and validation data, and they are stored on your disk like:
+
+### Data Extraction
+
+After you download the proper [ImageNet](http://www.image-net.org/index) archived files (the service needs registration) you will need to extract the images from the `.tar` files. You can use the `data/ilsvrc12/extract_ilsvrc.sh` script by specifing the path of the `.tar` files on the system at the `DATAPATH` variable and then executing:
+
+    ./data/ilsvrc12/extract_ilsvrc.sh
+
+We assume that you already have downloaded the ImageNet training data and validation data, and they are stored on your disk like (they should be if you used the `/data/ilsvrc12/extract_ilsvrc.sh` script):
 
     /path/to/imagenet/train/n01440764/n01440764_10026.JPEG
     /path/to/imagenet/val/ILSVRC2012_val_00000001.JPEG
 
-You will first need to prepare some auxiliary data for training. This data can be downloaded by:
+At this stage, the size of the data on your disk should be `140GB` for the (extracted) `train` images and `6.4GB` for the (extracted) `val` images.
+
+
+### Data Preprocessing
```

Comment:
Maybe someone can find a better name for that sub-header. The `Data Preprocessing` is good but it might not be the best. Any change is welcome.